### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by RAbraham** [mullr/micrologic#8]
@mullr I just wanted to thank you for one of the clearest explanations(actually, _the_ clearest explanation) of minikanren that I have read. As I implement my own, I have your page open every day.

**Comment by mullr** [comment]
:-D I'm happy that it helped you! I'll leave this issue open so you can post a link to your own implementation when it's done ;)

**Comment by dgr** [comment]
Indeed! I'll second that. Honestly, I've tried reading Byrd's thesis a couple of times and I always end up bewildered. But your explanation, @mullr ,  made it simple. Thanks!
